ui/src/api/index.js

```diff
@@ -38,3 +38,16 @@ export const getSurveys = () => {
 export const addSurvey = (data) => {
   return axios.post(ADD_SURVEY_URL, data);
 };
+
+const USER_SIGNUP_URL = "/authentication/signup";
+
+export const userSignup = (data) => {
+  let config = {
```

Comment:
This approach is good enough if we only had one or two APIs; however we have a lot of APIs and so I think it's better to make use of the default configuration... I don't test this code so it may not work, but it should be something like this:
```
axios.defaults.headers.common['Accept'] = 'application/json'; 
axios.defaults.headers.common['Content-Type'] = 'application/json'; 
```
Can you check to make sure whether the above settings work or not?

Reply:
This works. Will push my code soon.

Reply:
Great! When you worked on the login api, please also set the authentication in this way. I dont know how will the BE give the token, but when they did, please store it in vuex (if they havent stored already), so that we can use in the "Authentication" header XD but it is for later PR, not this one.